ticker/src/main/java/com/robinhood/ticker/TickerUtils.java

```diff
@@ -23,48 +23,13 @@
  * @author Jin Cao, Robinhood
  */
 public class TickerUtils {
-    public static final char EMPTY_CHAR = (char) 0;
-
-    /**
-     * @return a default ordering for all viewable ASCII characters. This list also special cases
-     *         space to be in front of the 0 and swap '.' with '/'. These special cases make
-     *         typical US currency animations intuitive.
-     */
-    public static char[] getDefaultListForUSCurrency() {
-        final int indexOf0 = (int) '0';
-        final int indexOfPeriod = (int) '.';
-        final int indexOfSlash = (int) '/';
-
-        final int start = 33;
-        final int end = 256;
-
-        final char[] charList = new char[end - start + 1];
-        for (int i = start; i < indexOf0; i++) {
-            charList[i - start] = (char) i;
-        }
-
-        // Special case EMPTY_CHAR and '.' <-> '/'
-        charList[indexOf0 - start] = EMPTY_CHAR;
-        charList[indexOfPeriod - start] = '/';
-        charList[indexOfSlash - start] = '.';
-
-        for (int i = indexOf0 + 1; i < end + 1; i++) {
-            charList[i - start] = (char) (i - 1);
-        }
+    static final char EMPTY_CHAR = (char) 0;
 
-        return charList;
+    public static char[] provideNumberList() {
+        return "0123456789".toCharArray();
     }
 
-    /**
-     * @return a default ordering for numbers.
-     */
-    public static char[] getDefaultNumberList() {
-        final char[] charList = new char[12];
-        charList[0] = EMPTY_CHAR;
-        charList[1] = (int) '.';
-        for (int i = 0; i < 10; i++) {
-            charList[i + 2] = (char) (i + 48);
-        }
-        return charList;
+    public static char[] provideAlphabeticalList() {
+        return "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ".toCharArray();
```

Comment:
won't this setup lead to some weird transitions from a capital letter to non-capital? like it has to cycle through the whole alphabet before it gets there?

probably a weird use-case, though, not sure i can think of a practical example.

Reply:
yeah if user wants to support that behavior, they can specify multiple lists to specify how transition between uppercase and lowercase should be animated.

FWIW our previous behavior was also the same thing since we were using ASCII ordering.